src/main/java/io/supertokens/httpRequest/HttpRequest.java

```diff
@@ -263,6 +262,31 @@ public static <T> T sendJsonPUTRequest(Main main, String requestID, String url,
         return sendJsonRequest(main, requestID, url, requestBody, connectionTimeoutMS, readTimeoutMS, version, "PUT");
     }
 
+    //TODO: tests!
```

Comment:
TODO here

Reply:
yeah. And also in the PR description. Should I remove it from the code?

Reply:
I didn't understand your comment.

Reply:
Oh yea, please do. It can be added in the checklist here: https://github.com/supertokens/supertokens-core/pull/1010

Reply:
okay, done